experimental/sdk/webpubsub-socketio-extension/src/SIO/components/web-pubsub-adapter.ts

```diff
@@ -1,14 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { debugModule } from "../../common/utils";
+import { WebPubSubServiceClient } from "@azure/web-pubsub";
 import { Packet } from "engine.io-parser";
```

Comment:
Is this `Packet` should come from `socket.io-parser`? Then `nsp` should in the interface

Reply:
updated